server/datastore/mysql/apple_mdm.go

```diff
@@ -1712,18 +1713,34 @@ func (ds *Datastore) bulkSetPendingMDMAppleHostProfilesDB(
 		( hmap.host_uuid IS NOT NULL AND ( hmap.operation_type = ? OR hmap.operation_type IS NULL ) )
 `, fmt.Sprintf(appleMDMProfilesDesiredStateQuery, "h.uuid IN (?)", "h.uuid IN (?)", "h.uuid IN (?)"))
 
-	// TODO: if a very large number (~65K) of host uuids was matched (via
-	// uuids, teams or profile IDs), could result in too many placeholders (not
-	// an immediate concern).
-	stmt, args, err := sqlx.In(toInstallStmt, uuids, uuids, uuids, fleet.MDMOperationTypeRemove)
-	if err != nil {
-		return ctxerr.Wrap(ctx, err, "building profiles to install statement")
+	selectProfilesBatchSize := 10_000
```

Comment:
TIL that underscore was a valid way to express numbers in Go.

How did you settle on 10K?

Reply:
great question! back of the envelope math, I ran the loadtest with this so that's why I decided to keep it. Should I add a comment?

Reply:
Sounds good! In particular, it'd be nice to have some callout with guidance for our future selves to know when we ought to adjust the batch size (e.g., when adding new parameters to the statement).